src/components/signup/constants.js

```diff
@@ -85,4 +93,58 @@ export const INPUTS = [
     },
 ];
 
+export const PRODUCTS = [
+    {
+        category: "Agrochemicals",
+        value: "agrochemicals",
+    },
+    {
+        category: "Beans",
+        value: "beans",
+    },
+    {    
+        category: "Fruits",
+        value: "fruits",
+    },
+    {
+        category: "Grains",
+        value: "grains",
+    },
+    {
+        category: "Nuts",
+        value: "nuts",
+    },
+    {
+        category: "Pesticides",
+        value: "pesticides",
+    },
+    {
+        category: "Plants",
+        value: "plants",
+    },
+    {
+        category: "Rice",
+        value: "rice",
+    },
+    {
+        category: "Seeds",
+        value: "seeds",
+    },
+    {
+        category: "Seedlings",
+        value: "seedlingss",
+    },
+    {
+        category: "Vegetable",
+        value: "vegetable",
+    },
+    {
+        category: "Wheat",
+        value: "wheat",
+    },
+    {
+        category: "Other",
+        value: "other",
+    },
+];
```

Comment:
good use of object of arrays here. You are starting to get it

Reply:
Thanks